docs/docs/integrations/tools/goat.ipynb

```diff
@@ -0,0 +1,217 @@
+    "\n",
+    "- **By use case**\n",
+    "  - **Money transmission**\n",
+    "    - Send and receive payments [[EVM](https://github.com/goat-sdk/goat/tree/main/python/examples/by-use-case/evm-send-and-receive-tokens), [Solana](https://github.com/goat-sdk/goat/tree/main/python/examples/by-use-case/solana-send-and-receive-tokens)]\n",
+    "  - **Investing**\n",
+    "    - Generate yield [[Solana](https://github.com/goat-sdk/goat/tree/main/python/examples/by-use-case/solana-usdc-yield-deposit)]\n",
+    "    - Purchase crypto assets [[EVM](https://github.com/goat-sdk/goat/tree/main/python/examples/by-use-case/evm-swap-tokens), [Solana](https://github.com/goat-sdk/goat/tree/main/python/examples/by-use-case/solana-swap-tokens)]\n",
+    "- **By wallet**\n",
+    "  - [Crossmint](https://github.com/goat-sdk/goat/tree/main/python/examples/by-wallet/crossmint)\n",
+    "- **See all python quickstarts [here](https://github.com/goat-sdk/goat/tree/main/python/examples).**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "abd26764",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "1. Install the core package and langchain adapter:\n",
+    "\n",
+    "```bash\n",
+    "pip install goat-sdk goat-sdk-adapter-langchain\n",
+    "```\n",
+    "\n",
+    "2. Install the type of wallet you want to use (e.g solana):\n",
+    "\n",
+    "```bash\n",
+    "pip install goat-sdk-wallet-solana\n",
+    "```\n",
+    "\n",
+    "3. Install the plugins you want to use in that chain:\n",
+    "\n",
+    "```bash\n",
+    "pip install goat-sdk-plugin-spl-token\n",
+    "```\n",
+    "\n",
+    "## Instantiation\n",
+    "\n",
+    "Now we can instantiate our toolkit:\n",
+    "\n",
+    "```python\n",
+    "from goat_adapters.langchain import get_on_chain_tools\n",
+    "from goat_wallets.solana import solana, send_solana\n",
+    "from goat_plugins.spl_token import spl_token, SplTokenPluginOptions\n",
+    "from goat_plugins.spl_token.tokens import SPL_TOKENS\n",
+    "\n",
+    "# Initialize Solana client\n",
+    "client = SolanaClient(os.getenv(\"SOLANA_RPC_ENDPOINT\"))\n",
+    "\n",
+    "# Initialize regular Solana wallet\n",
+    "keypair = Keypair.from_base58_string(os.getenv(\"SOLANA_WALLET_SEED\") or \"\")\n",
+    "wallet = solana(client, keypair)\n",
+    "\n",
+    "tools = get_on_chain_tools(\n",
+    "        wallet=wallet,\n",
+    "        plugins=[\n",
+    "            send_solana(),\n",
+    "            spl_token(SplTokenPluginOptions(\n",
+    "                network=\"mainnet\",  # Using devnet as specified in .env\n",
+    "                tokens=SPL_TOKENS\n",
+    "            )),\n",
+    "        ],\n",
+    "    )\n",
+    "```\n",
+    "\n",
+    "## Invocation\n",
+    "```python\n",
+    "tools[\"get_balance\"].invoke({ \"address\": \"0x1234567890123456789012345678901234567890\" })\n",
+    "```\n",
+    "\n",
+    "## Use within an agent\n",
```

Comment:
Why not show the only the tool? The scaffolding that shows main() or instantiation of a chat model seems unnecessary? This doc isn't meant to teach users how to use agents or chat models, but show an example of how to instantiate and create the tool.

Reply:
Agree! Though the lint fails if you dont add a "Use within agent" section. Should I remove it anyway?

Reply:
I see it's part of the linter then. OK keep as is for now. We'll do a separate pass to fix this.

Reply:
Could you follow up on this PR w/ an example of how to do this with the prebuilt langgraph agent? I just want to unblock this PR to reduce WIP

Reply:
ah you'll need to resolve the conflict as well on packages.yml -- if you insert it alphabetically it'll reduce merge conflicts likelihood

Reply:
Can do! Fixed the conflicts too! Thank you! :)